ibis/backends/conftest.py

```diff
@@ -461,6 +464,31 @@ def lock_load_data(
     return cls(data_directory)
 
 
+def pytest_sessionfinish(session, exitstatus):
```

Comment:
For reference to anyone curious. I have never heard of this: https://docs.pytest.org/en/7.1.x/_modules/_pytest/hookspec.html#pytest_sessionfinish